vault/testing.go

```diff
@@ -2041,7 +2042,7 @@ func (tc *TestCluster) initCores(t testing.T, opts *TestClusterOptions, addAudit
 			"path":        "secret/",
 			"description": "key/value secret storage",
 			"options": map[string]string{
-				"version": "1",
+				"version": opts.KVVersion,
```

Comment:
It looks like `opts.KVVersion` in initCores is causing a panic in some tests that use `initCores` because there's no default. Could you please add a default value?

Reply:
done